README_PPX.md

```diff
@@ -34,36 +34,36 @@ extension point.
 
 ### Installation and usage
 
-`ppx_irmin` may be installed via [opam](https://opam.ocaml.org/):
+`ppx_repr` may be installed via [opam](https://opam.ocaml.org/):
 
 ```
-opam install ppx_irmin
+opam install ppx_repr
 ```
 
 If you're using the [dune](https://github.com/ocaml/dune) build system, add the
 following field to your `library`, `executable` or `test` stanza:
 
 ```
-(preprocess (pps ppx_irmin))
+(preprocess (pps ppx_repr))
 ```
 
-You can now use `[@@deriving irmin]` after a type declaration in your code to
-automatically derive an Irmin type representation with the same name.
+You can now use `[@@deriving repr]` after a type declaration in your code to
+automatically derive a type representation with the same name.
 
 ### Specifics
 
-`ppx_irmin` supports all of the type combinators exposed in the
-[Irmin.Type](https://docs.mirage.io/irmin/Irmin/Type/index.html) module (basic
+`ppx_repr` supports all of the type combinators exposed in the
+[Repr](https://docs.mirage.io/irmin/Irmin.Type/index.html) module (basic
```

Comment:
This is still linking to Irmin.Type

Reply:
True, but there is no correct place for it to point to yet... I'll change it to `repr/Repr/index.html` in anticipation of an eventual release there.